packages/getlit-sdk/src/index.ts

```diff
@@ -0,0 +1,145 @@
+import './global';
+import { LitOptionsBuilder } from './lib/lit-options-builder';
+import { isBrowser, log } from './lib/utils';
+import { handleAutoAuth } from './lib/auth/handle-auto-auth';
+import { LitEmitter } from './lib/events/lit-emitter';
+import { LitStorage } from '@lit-protocol/lit-storage';
+import { AuthMethod } from '@lit-protocol/types';
+import { HeliaProvider } from './lib/ipfs-provider-sdk/providers/helia-provider';
+import { BaseIPFSProvider } from './lib/ipfs-provider-sdk/providers/BaseIPFSProvider';
+import { PinataProvider } from './lib/ipfs-provider-sdk/providers/pinata-provider';
+import {
+  PersistentStorageConfig,
+  PersistentStorageConfigOptions,
+  infuraConfig,
+  pinataConfig,
+} from './lib/types';
+import { infuraProvider } from './lib/ipfs-provider-sdk/providers/infura-provider';
+
+// initialize globally
+export const loadLit = async ({
+  debug = true,
+  persistentStorage = {
```

Comment:
We should allow configuring this on the `lit-option-builder`instead of default parameters decelared here.